src/components/Layout/Header.tsx

```diff
@@ -0,0 +1,52 @@
+import Logo from "../../assets/images/logo/logo.svg";
+import { Link, NavLink, useLocation } from "react-router-dom";
+import "./Header.css";
+
+// react-router-dom의 NavLink를 이용하면 활성화된 네비게이션 항목을 하이라이트해줄 수 있어요!
+function getLinkStyle({ isActive }: { isActive: boolean }) {
```

Comment:
```suggestion
function getLinkStyle(isActive: boolean) {
```
으로 간결하게 쓸 수 있을 거 같아요!